aws-datastore/src/androidTest/java/com/amplifyframework/datastore/BasicCloudSyncInstrumentationTest.java

```diff
@@ -103,17 +100,35 @@ public void setup() throws AmplifyException {
         api = SynchronousApi.delegatingTo(apiCategory);
         appSync = SynchronousAppSync.using(AppSyncClient.via(apiCategory));
 
+        long tenMinutesAgo = new Date().getTime() - TimeUnit.MINUTES.toMillis(10);
+        Temporal.DateTime tenMinutesAgoDateTime = new Temporal.DateTime(new Date(tenMinutesAgo), 0);
         DataStoreCategory dataStoreCategory = DataStoreCategoryConfigurator.begin()
             .api(apiCategory)
-            .clearDatabase(true)
             .context(context)
             .modelProvider(AmplifyModelProvider.getInstance())
             .resourceId(configResourceId)
             .timeout(TIMEOUT_SECONDS, TimeUnit.SECONDS)
+            .dataStoreConfiguration(DataStoreConfiguration.builder()
+                .syncExpression(BlogOwner.class, () -> BlogOwner.CREATED_AT.gt(tenMinutesAgoDateTime))
```

Comment:
I assume that existing records in DynamoDB that don't have createdAt are excluded from the result right?

Reply:
Great question.  Generally, I think the answer is yes - a null value would probably not match a greater than filter.  However, I don't think it matters because all the records should actually have values for `createdAt`.  This field has been implicitly created by AppSync for a while now.  This PR just makes it an explicitly defined field on the model, so that it can be used in a sync expression.